src/public/services/AdminFormService.ts

```diff
@@ -0,0 +1,32 @@
+import axios from 'axios'
+
+const ADMIN_FORM_ENDPOINT = '/api/v1/admin/form'
```

Comment:
I'm thinking we could start numbering the APIs from v3 instead for consistency, as the submission endpoints are already on v2.

Reply:
we should also set up a hierarchy of routers for each `/`

Reply:
Ya, this is client side, on backend there is 

ApiRouter `/api/v3` -> AdminRouter `/admin` -> individual endpoints like update form settings `/form/settings`

Reply:
Maybe I'll nest another AdminFormRouter for the `/form` endpoint?

Reply:
No, I think you should do it to reflect the hierarchy of the routes. Many projects even have their folder structure organised that way.

Reply:
Done in 00caf9f